exporter/datadogexporter/logs_exporter.go

```diff
@@ -57,61 +57,6 @@ type logsExporter struct {
 	gatewayUsage     *attributes.GatewayUsage
 }
 
-// newLogsExporter creates a new instance of logsExporter
```

Comment:
I believe everything in this file can be removed except for `newLogsAgentExporter` and the constants

Reply:
thanks, removed 👍